src/redux/books/books.js

```diff
@@ -19,7 +37,7 @@ const bookReducer = (state = initialState, action) => {
         { title: action.payload.title, author: action.payload.author },
       ];
     case REMOVE_BOOK:
-      return state.filter((book) => book.id !== action.payload);
+      return state.filter((book) => book.title !== action.payload.title);
```

Comment:
- [ ] Kindly fix the issue with the `id`s as requested by the previous reviewer, every book should have a unique id and the books removal should be done by `id` and not by `title` since 2 books could have the same title 😅